woocommerce/woocommerce-android
# Shipping Labels: enable customs feature and update tracking

Closes #4246, this PR contains two main changes:
1. Enables the customs feature in the shipping labels form.
2. Updates the Shipping Label tracking:
     - It removes most of the redundant button tapping events, as they are tracked by the step completion steps.
     - It adds new states to the `shipping_labels_purchase_flow` event to represent starting steps, this replaces both the "continue" and "edit" button tap events.
     - I kept the address related events, as they can't be easily tracked using the `shipping_labels_purchase_flow`, and I updated the spreadsheet to include them (https://d.pr/i/dX7Vsk)
3. Updates the "reprint shipping label" CTA text to just "print shipping label" (internal ref: p1623243768209200/1623243663.209100-slack-CGPNUU63E), I didn't want to open a separate PR just to update some text.

#### Testing
1. Create an international order in your store.
2. Open the order in the app.
3. Confirm that you can see the "create shipping label" button.
4. proceed to shipping label creation.
5. Test the customs form and confirm that it works.
6. Confirm that shipping label creation works.
7. Check the event `shipping_labels_purchase_flow` in Tracks, and confirm that it's sent for all steps.

Update release notes:
